Pipfile

```diff
@@ -18,6 +18,7 @@ html2text = "*"
 [dev-packages]
 graphviz = "==0.19.1"
 pytest = "==7.1.1"
+black = "*"
```

Comment:
Can we add some docs to the repo for how we should be using black?

Reply:
I added a section to the README, though I may update it based on whether we want to do a black check on the PR build.  I think we should.